vaadin-board-row.html

```diff
@@ -55,20 +55,26 @@
        * Calculates flex basis based on colSpan and width
        * @param{Number} colspan value of the row
        * @param{Number} width of the row in px
-       * @param{Number} amount of nodes in the row
+       * @param{Number} amount of columns in the row
        */
-      _calculateFlexBasis(colSpan, width, nodesCount) {
-        let flexBasis = 25 * colSpan;
+      _calculateFlexBasis(colSpan, width, colsInRow) {
```

Comment:
Maybe an easier way to write the whole _calculateFlexBasis:

```
_calculateFlexBasis(colSpan, width, colsInRow) {
  if (width < this._ONE_COLUMN_MAX_WIDTH) {
    colsInRow = 1;
  } else if (width < this._TWO_COLUMNS_MAX_WIDTH && colsInRow == 4) {
    colsInRow = 2;
  }
  let flexBasis = colSpan / colsInRow * 100;
  flexBasis = flexBasis > 100 ? 100 : flexBasis;
  return flexBasis + '%';
}
```
Basically - if in "small" size, always use one column, if in "medium size" and you have 4 columns, use 2 columns, otherwise calculate with the basic formula.